src/components/dev-hub/icons/star-icon.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useTheme } from 'emotion-theming';
```

Comment:
The package name for this has recently changed! I would recommend pulling in the latest from the branch and running `npm ci` to see it on your end.

```suggestion
import { useTheme } from '@emotion/react';
```